_config.yml

```diff
@@ -17,9 +17,9 @@ linkedin_username: noah-hilt-1b68991a
 # about_title: About Me (Use this to override about section title)
 about_profile_image: images/profile.jpg
 about_content: | # this will include new lines to allow paragraphs
-  Hi 👋 my name is Noah and I've been an iOS developer for the past 9 years. I have a wide range of experience in every facet of app development, building from the ground up or refactoring ancient codebases. I am a strong believer in pragmatic programming and bringing the user experience to the forefront.
-  
-  My journey began with `C++` → `Objective-C` → `Swift`
+  Hi 👋, my name is Noah, and I've been an iOS developer for the past nine years. I have a wide range of experience in every facet of app development, building from the ground up or refactoring ancient codebases. I am a strong believer in pragmatic programming and bringing the user experience to the forefront.
+
+  My progression of programming langauges is `C++` → `Objective-C` → `Swift`.
```

Comment:
I reworked this sentence because, contrary to `upstream`'s implication, the journey did not begin with Objective-C or Swift.

Reply:
I agree that does make more sense, although 'progression of programming languages' sounds a little too systematic to me?

Reply:
also a small typo `langauges` -> `languages` 👌